seedServer/src/main/java/facades/UserFacade.java

```diff
@@ -47,6 +48,22 @@ public IUser getUserByUserId(String id) {
           em.close();
       }
   }
+  
+  public void setUser(String username, String password) throws PasswordStorage.CannotPerformOperationException{
```

Comment:
Please rename this to registerUser, because setUser sounds like a setter which this is not.

Reply:
Done